jsk_baxter_robot/jsk_baxter_startup/baxter.launch

```diff
@@ -23,7 +23,6 @@
        doc="Baxter arm controller mode: positon_w_id, position or velocity (default: position_w_id)" />
   <arg name="USER_NAME" default="false"/>
   <param name="/active_user/launch_user_name" value="$(arg USER_NAME)"/>
-  <machine name="localhost" address="localhost" env-loader="/opt/ros/indigo/env.sh" user="baxter"/>
```

Comment:
MEMO: I approve this change because I couldn't understand why this was needed.
The commit where this was added: https://github.com/jsk-ros-pkg/jsk_robot/commit/ca24d03d436f5debe9f5117252b1ca30db5bcd65
At that time, `localhost` was used in [baxter_tweet.launch](https://github.com/jsk-ros-pkg/jsk_robot/blob/0.0.4/jsk_baxter_robot/jsk_baxter_startup/jsk_baxter_lifelog/baxter_tweet.launch) and [tweet.launch](https://github.com/jsk-ros-pkg/jsk_robot/blob/0.0.4/jsk_robot_common/jsk_robot_startup/lifelog/tweet.launch).